database/seeders/LanguageSeeder.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+use Illuminate\Database\Seeder;
+
+class LanguageSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        $languages = ['Castellà', 'Català', 'Anglès', 'Francès', 'Alemany', 'Italià', 'Portuguès', 'Rus'];
+        $levels = ['Bàsic', 'Intermedi', 'Avançat', 'Natiu'];
+
+        foreach ($languages as $language) {
+            foreach ($levels as $level) {
+                DB::table('languages')->insert([
```

Comment:
I like this method, really well thought! But, I'm pretty sure we can create and use a factory here instead of use the DB method. Then the factory could be useful when if we want to create different situations.

Reply:
I will try to work on it!!

Reply:
Perhaps this suggestion should be addressed later, as I haven't been able to implement it in a way that the seeder does what I currently need.